tests/raftstore/node.rs

```diff
@@ -102,8 +115,18 @@ impl Channel<RaftMessage> for ChannelTransport {
                 core.snap_paths[&to_store].0.wl().deregister(&key, &SnapEntry::Receiving);
             });
 
-            if !dst_file.exists() {
-                try!(fs::copy(source_file.path(), dst_file.path()));
+            if !writer.exists() {
+                let buf_len = 10000;
```

Comment:
It's better to use the read/write interfaces rather than copy the underlying files, then it's ready to add the new format snapshot.